test/server/controllers/ArticleController.test.js

```diff
@@ -0,0 +1,136 @@
+import chaiHttp from 'chai-http';
+import chai from 'chai';
+import app from '../../../app';
+import {
+  articleInputValid,
+  articleInputNoTitle,
+} from '../../../mockdata/articleMockData';
+
+chai.should();
+
+chai.use(chaiHttp);
+
+describe('Articles Controller Tests', () => {
+  const userData = {};
+  before((done) => {
+    // Sign up a user and get the id and token returned
+    chai.request(app)
+      .post('/api/v1/users/signup')
+      .send({
+        fullName: 'Not Tani',
+        email: 'n.tani@whowa.com',
+        password: 'ntanirfsee4',
+      })
```

Comment:
Hmmmm, I see what you mean. However, don't you think this will break once we start using uuid as the user primary key? since I would have to specify the id going into the createToken function right? I was considering that when writing this, before we even introduced the user seeders.

Reply:
Maybe what I could do is make the userData object get the id and token from the other sign up test written in this same file, what do you think of this?

Reply:
Makes sense about the UUID and the last solution you suggested. Awesome